src/sonic-yang-models/yang-models/sonic-fabric-port.yang

```diff
@@ -0,0 +1,54 @@
+module sonic-fabric-port{
+
+    yang-version 1.1;
+
+    namespace "http://github.com/sonic-net/sonic-fabric-port";
+    prefix fabric-port;
+
+    description "FABRIC_PORT yang Module for SONiC OS";
+
+    revision 2023-03-14 {
+        description "First Revision";
+    }
+
+    container sonic-fabric-port {
+
+        container FABRIC_PORT {
+
+            description "FABRIC_PORT part of config_db.json";
+
+            list FABRIC_PORT_LIST {
+
+                key "name";
+
+                leaf name {
+                    type string {
+                        length 1..128;
+                    }
+                }
+
+                leaf isolateStatus {
+                    type string {
+                        pattern "False|True";
+                    }
+                }
+
+                leaf alias {
+                    type string {
+                        length 1..128;
+                    }
+                }
+
+                leaf lanes {
```

Comment:
I think, the fabric port configuration without specifying `lanes` is not valid, can we add this field as `mandatory`

Reply:
fixed.